Tests/Acceptance/DoctrineAccessTokenManagerTest.php

```diff
@@ -52,16 +52,16 @@ public function testClearExpired(): void
     private function buildClearExpiredTestData(Client $client): array
     {
         $validAccessTokens = [
-            $this->buildAccessToken('1111', '+1 day', $client),
-            $this->buildAccessToken('2222', '+1 hour', $client),
-            $this->buildAccessToken('3333', '+1 second', $client),
-            $this->buildAccessToken('4444', 'now', $client),
+            $this->buildAccessToken($client, '1111', '+1 day'),
```

Comment:
@franjo-zadelj-trikoder What is the purpose of the signature change? As far as I can see it is not needed and just adds noise to the diff.

Reply:
was changed in the previous version when it was completely different logic, so it was left as it was. reverted to the initial state.